elements/diff/nuxeo-versions-diff-button.html

```diff
@@ -71,6 +71,11 @@
 
       _doDiff: function() {
         this.$$('#opGetVersions').execute().then(function(result) {
+          // check if there is at least two versions to be compared
+          if (result.entries.length < 2) {
+            this.fire('notify', {message: this.i18n('versions.nothingToCompare')});
```

Comment:
We usually make the first part of the label the name of the current element in camelCase:
```suggestion
            this.fire('notify', { message: this.i18n('versionsDiffButton.nothingToCompare') });
```